tests/dictionary.test.ts

```diff
@@ -12,8 +13,6 @@ const index = {
   uid: 'movies_test',
 };
 
-jest.setTimeout(100 * 1000);
```

Comment:
This is replaced by a global setting in `vitest.config.js` `testTimeout: 100_000, // 100 seconds`.